public/index.html

```diff
@@ -8,7 +8,7 @@
     <title>Network Canvas</title>
   </head>
   <body>
-    <script type="text/javascript" src="cordova.js"></script>
+    <!-- <script type="text/javascript" src="cordova.js"></script> -->
```

Comment:
This is added automatically by the Cordova build process. It can be removed from outside of `/www`.